software/src/meson.build

```diff
@@ -10,3 +10,5 @@ subdir('backends')
 project_sources += files(
     'pcie.cpp'
 )
+
+install_headers(files('pcie.h'), subdir: 'enso')
```

Comment:
`pcie.h` was hidden on purpose. Do you need it from shinkansen?

Reply:
I just checked shinkansen and I see why now. My worry is that once we expose this, it becomes part of the API and pcie.h is not particularly well thought out.